File: snippets/node/resume_sending_from_a_paused_subaccount.js
const mailchimp = require("mailchimp-transactional")("YOUR_API_KEY");

async function run() {
  const response = await mailchimp.subaccounts.resume({
    id: "UNIQUE_ACCOUNT_ID"
  });
  console.log(`This subaccount is now ${response.status}`);
}

run();

File: snippets/ruby/resume_sending_from_a_paused_subaccount.rb
require 'MailchimpTransactional'

client = MailchimpTransactional::Client.new(ENV['YOUR_API_KEY'])
account_id = "UNIQUE_ACCOUNT_ID"

begin
  account = client.subaccounts.resume(id: account_id)
  puts account["status"] # => “active”
rescue MailchimpTransactional::ApiError => e
  puts "Error: #{e}"
end